gempak/ush/gfs_meta_sa2.sh

```diff
@@ -362,26 +326,31 @@ HILO    = 31/H#;L#/1020-1060;900-1010/5/10;10!2/H#;L#/1020-1060;900-1010/5/10;10
 TITLE   = 31/-1/~ ? ${MDL} PMSL (WHITE)|~UK VS ${MDL} PMSL!2/-2/~ ? UKMET PMSL (RED)
 l
 r
- 
+
 ex
 EOF25
 done
-
-
-if [ $SENDCOM = "YES" ] ; then
-   mv ${metaname} ${COMOUT}/${mdl}_${PDY}_${cyc}_${metatype}
-   if [ $SENDDBN = "YES" ] ; then
-      ${DBNROOT}/bin/dbn_alert MODEL ${DBN_ALERT_TYPE} $job \
-      ${COMOUT}/${mdl}_${PDY}_${cyc}_${metatype}
-      if [ $DBN_ALERT_TYPE = "GFS_METAFILE_LAST" ] ; then
-        DBN_ALERT_TYPE=GFS_METAFILE
-        ${DBNROOT}/bin/dbn_alert MODEL ${DBN_ALERT_TYPE} $job \
-        ${COMOUT}/${mdl}_${PDY}_${cyc}_${metatype}
-      fi
-   fi
+export err=$?;err_chk
+
+#####################################################
+# GEMPAK DOES NOT ALWAYS HAVE A NON ZERO RETURN CODE
+# WHEN IT CAN NOT PRODUCE THE DESIRED GRID.  CHECK
+# FOR THIS CASE HERE.
+#####################################################
+if (( err != 0 )) || [[ ! -s "${metaname}" ]] &> /dev/null; then
+    echo "FATAL ERROR: Failed to create gempak meta file ${metaname}"
+    exit $(( err + 100 ))
 fi
 
-#export COMIN=/com/nawips/${envir}/ukmet.${PDY}
-echo " end with ukmet_gfs_meta_sa2.sh"
+mv "${metaname}" "${COM_ATMOS_GEMPAK_META}/${mdl}_${PDY}_${cyc}_${metatype}"
```

Comment:
There is no gfs_20240314_00_sa2 file under /lfs/h1/ops/prod/com/gfs/v16.3/gfs.20240314/00/atmos/gempak/meta. So, similar to the gdas_ecmwf_meta_ver.sh and gdas_ukmet_meta_ver.sh scripts, this may be a leftover script that no longer needed.

Reply:
This file is also generated for the 06 cycle only. It is located at /lfs/h1/ops/prod/com/gfs/v16.3/gfs.20240315/06/atmos/gempak/meta.